packages/sdk/src/client/edition/schedules.ts

```diff
@@ -122,12 +123,14 @@ export async function editionRegisteredMinters(
   // Check supportsInterface() to verify each address is a minter
   const minters = await Promise.all(
     candidateMinters.map(async (minterAddress) => {
-      const minterContract = IMinterModule__factory.connect(minterAddress, signerOrProvider)
+      const minterContract = IMinterModuleV2__factory.connect(minterAddress, signerOrProvider)
 
       try {
-        const isMinter = await minterContract.supportsInterface(interfaceIds.IMinterModule)
+        const moduleInterfaceId = await minterContract.moduleInterfaceId()
```

Comment:
we have multiple valid interfaces now, IMinterModule and IMinterModuleV2

we can use moduleInterfaceId which exists in both interfaces and check that it matches a known interface and do this in a single call